src/main/resources/tomcat-eureka-config/dbmigration/registry-changelog-ec-user-config.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!--
+  #%L
+  Eureka! Clinical Registry Service
+  %%
+  Copyright (C) 2017 - 2018 Emory University
+  %%
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+       http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  #L%
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                         http://www.liquibase.org/xml/ns/dbchangelog
+                         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd"
+		   logicalFilePath="changelog-master.xml">
+    <changeSet author="arpost" id="EC-USER-1">
+	<insert tableName="COMPONENTS">
+	    <column name="ID" valueSequenceNext="COMPONENT_SEQ"/>
+	    <column name="NAME" value="Eureka! Clinical User Service"/>
+	    <column name="DISPLAYNAME" value="User Service"/>
+	    <column name="DESCRIPTION" value="User portal and profile management"/>
+	    <column name="URL" value="{{ec_user_service_base_url}}"/>
```

Comment:
All of the URLs for the registry entries need to be syntactically valid URLs, even if not all of these services will be running in embedded tomcat.